plugin/pkg/scheduler/algorithm/priorities/priorities.go

```diff
@@ -34,7 +34,7 @@ func calculateScore(requested int64, capacity int64, node string) int {
 		return 0
 	}
 	if requested > capacity {
-		glog.Infof("Combined requested resources %d from existing pods exceeds capacity %d on node %s",
+		glog.V(4).Infof("Combined requested resources %d from existing pods exceeds capacity %d on node %s",
```

Comment:
I'm torn between if this should be ```V(3)``` (Extended information about changes) or ```V(4)``` (Debug level verbosity). I veer towards ```V(3)``` as it's extended information even though it's about why the pod didn't land on a node.